### PR TITLE
change indent

### LINKED ISSUE #1821: Indent in json files
In this project a lot of json files are used.

In the `user-agents` and the `core` directories these files have an indent of 2 spaces.
In the `browsers` and `devices` directories these files have an indent of 4 spaces.

The json files should use the same indent all over the project. Which one is prefered?

### PR DESCRIPTION
**Pull request by mimmi20**
This PR changes the indent in the json files to 2 spaces.

This PR also adds commands to rewrite or validate different json files. Most files are validated or rewritten against a json schema. To rewrite the files in the `user-agents` folder Twig is used (The schema can not handle our style to write the values in arrays).

This PR requires PR #1826 to be merged before.

Fixes #1821.

### CODE REVIEW

File: resources/user-agents/browsers/k-ninja/k-ninja-generic.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
           "match": "Mozilla/5.0 (#PLATFORM#) Gecko/* K-Ninja/*",
           "platforms": [
             "WinXPa_32",
+            "Win98_2", "Win98_3",
             "Windows",
-            "Win98_3",
-            "Win98_2", "Win98"
+            "Win98"
```
Comment (jaydiablo) [inline resources/user-agents/browsers/k-ninja/k-ninja-generic.json]: Shouldn't `Win98` go up with the other two, and `Windows` be last?

File: src/Browscap/Command/RewriteDivisionsCommand.php [hunk 2]
```diff
@@ -0,0 +1,934 @@
...
+    /**
+     * @param array $platforms
+     *
+     * @return array
+     */
+    private function sortPlatforms(array $platforms)
+    {
+        $platformVersions = [];
+
+        $allplatforms = [
```
Comment (jaydiablo) [inline src/Browscap/Command/RewriteDivisionsCommand.php]: Is it possible to somehow use the `platforms.json` file for this listing, and maintain the sorting of the platforms in that file?

Seems like a bit of a pain if we have to update this list whenever a new platform is added/removed/changed.

File: resources/user-agents/browsers/k-ninja/k-ninja-generic.json (hunk 1 above)
Comment (mimmi20) [reply resources/user-agents/browsers/k-ninja/k-ninja-generic.json]: No. `Win98` matches to `*Win98*` while `Windows` matches to `*Windows*`.

File: src/Browscap/Command/RewriteDivisionsCommand.php (hunk 2 above)
Comment (mimmi20) [reply src/Browscap/Command/RewriteDivisionsCommand.php]: Will try to change this.

File: resources/user-agents/apps/snapchat/snapchat-10-13-to-10-16.json [hunk 3]
```diff
@@ -23,7 +23,10 @@
             "iPhone": "iPhone"
           },
           "platforms": [
-            "iOS_E_10_3", "iOS_E_10_2", "iOS_E_10_1", "iOS_E_10_0",
+            "iOS_E_10_0",
+            "iOS_E_10_1",
+            "iOS_E_10_2",
+            "iOS_E_10_3",
```
[the remark this answers is not in the document]
Comment (mimmi20) [reply resources/user-agents/apps/snapchat/snapchat-10-13-to-10-16.json]: This is an error by sorting. I have to fix this.

[the remark this answers is not in the document]
Comment (mimmi20) [reply resources/user-agents/apps/snapchat/snapchat-10-13-to-10-16.json]: fixed

**Pull request merged**